### PR TITLE
enhance(oidc): add more claims

### PR DESCRIPTION
**Pull request by JordanSussman**
closes https://github.com/go-vela/community/issues/996

Note: I intended to include whether the build was triggered from a fork, but it seems we don’t currently store that information in the database. Therefore, I decided to omit that detail for now. If someone wants to add this feature in the future, they will also need to update the database to capture this data.

### CODE REVIEW

File: mock/server/authentication.go [hunk 1]
```diff
@@ -40,6 +41,8 @@ const (
     "actor_scm_id",
     "commands",
     "image",
+	"image_name",
+	"image_tag",
```
Comment (wass3r) [inline mock/server/authentication.go:45]: looks like these are not indented right?
image

**Review by wass3r — COMMENTED** [review]
one minor thing, otherwise lgtm

**Comment by ecrupper** [comment]
Is there an intention to follow up this PR with the API implementation [here](https://github.com/go-vela/server/blob/0894432bab40b5b6981007233aca0f7d093194b1/api/build/id_token.go#L76-L97)?

File: mock/server/authentication.go (hunk 1 above)
Comment (JordanSussman) [reply mock/server/authentication.go]: This should be resolved now.

**Comment by JordanSussman** [comment]
> Is there an intention to follow up this PR with the API implementation [here](https://github.com/go-vela/server/blob/0894432bab40b5b6981007233aca0f7d093194b1/api/build/id_token.go#L76-L97)?

Can you expand on this a bit more? The code I added should be returned in the API payload due to https://github.com/go-vela/server/blob/0894432bab40b5b6981007233aca0f7d093194b1/api/build/id_token.go#L121.

**Comment by ecrupper** [comment]
> > Is there an intention to follow up this PR with the API implementation [here](https://github.com/go-vela/server/blob/0894432bab40b5b6981007233aca0f7d093194b1/api/build/id_token.go#L76-L97)?
> 
> Can you expand on this a bit more? The code I added should be returned in the API payload due to
> 
> https://github.com/go-vela/server/blob/0894432bab40b5b6981007233aca0f7d093194b1/api/build/id_token.go#L121
> 
> .

Whoops, got confused and thought `image` was net new. Yep lgtm

**Review by ecrupper — APPROVED** [review]

**Review by wass3r — APPROVED** [review]

**Review by plyr4 — APPROVED** [review]

**Pull request merged**